Update getting started (#385)
aicoe-aiops/ocp-ci-analysis: docs/kubernetes-contributor-get-started.md

# Engage as a Kubernetes Contributor

> If you are a Kubernetes contributor and would like to engage with the [AI4CI](../README.md) project, here are a few ways to get started and we compile a list of ways you can engage with this project.

Reviewer: ```suggestion
If you are a Kubernetes contributor and would like to engage with the AI4CI project, here are a few ways to get started.
```

## Kubernetes CI/CD Data Sources

### Analyzing TestGrid

> In order to quantify the current state of the CI workflow and identify any gaps within the CI process better, we calculate certain key performance indicator metrics related to the test runs using the data obtained from [TestGrid](https://testgrid.k8s.io/).

Reviewer: ```suggestion
In order to quantify the current state of the CI workflow and identify any gaps within the CI process, we calculate certain key performance indicators and metrics related to the test runs using the data obtained from [TestGrid](https://testgrid.k8s.io/).
```

> Here is a short [demo](https://youtu.be/4Mlg2qijEgQ) on how we extract data from the testgrid UI and calculate metrics from it and display them on dashboards.

Reviewer: ```suggestion
Here is a short [demo](https://youtu.be/4Mlg2qijEgQ) on how we extract data from the testgrid UI, calculate metrics from it and display them on dashboards.
```

You can find links to various notebooks and videos describing analyses done on Testgrid data [here](content.md#testgrid).

### Analyzing Prow

> [Prow](https://prow.k8s.io/) logs represent a rich source of information for automated triaging and root cause analysis. But unfortunately, these logs are noisy data types, i.e. two logs of the same kind but from two different sources may be different enough at a character level that traditional comparison methods are insufficient to capture this similarity. To overcome this issue, we use the Prow logs to learn log templates that denoise log data and help improve performance on downstream ML tasks.

Reviewer: I don't think we currently "use the Prow logs to learn log templates that denoise log data and help improve performance on downstream ML tasks"

Author: Makes sense, fixed that :+1:
